plasmapy/analysis/nullpoint.py

```diff
@@ -1154,18 +1155,57 @@ def in_bound(pos):
     return null_point_find(**null_point_args)
 
 
+def _vspace_iterator(vspace, MAX_ITERATIONS=500, err=1e-10):
```

Comment:
```suggestion
def _vspace_iterator(vspace, maxiter=500, err=1e-10):
```
I'm thinking it'd be worth using `maxiter` here because that's the parameter name that is used most often in SciPy for the maximum number of iterations.